src/Endpoints/Delegates/HandlesSettings.php

```diff
@@ -361,4 +361,21 @@ public function resetProximityPrecision(): array
     {
         return $this->http->delete(self::PATH.'/'.$this->uid.'/settings/proximity-precision');
     }
+
+    // Settings - Experimental: Embedders (hybrid search)
+
+    public function getEmbedders(): array
+    {
+        return $this->http->get(self::PATH.'/'.$this->uid.'/settings/embedders');
+    }
+
+    public function updateEmbedders(array $embedders): array
```

Comment:
`$embedders` structure should be defined

Reply:
@norkunas we don't have the time to do it for this PR, can we open a PR for the community to update it?

Reply:
yup, but I don't find anywhere docs about this :)

Reply:
@curquiza from https://www.meilisearch.com/docs/learn/experimental/vector_search and https://github.com/meilisearch/meilisearch/releases/tag/v1.6.0 I understand that structure should be like: `array<non-empty-string, array{source: non-empty-string, apiKey?: non-empty-string, model?: non-empty-string, documentTemplate?: non-empty-string, dimensions: positive-int}>` can you confirm?